lstchain/image/muon/muon_analysis.py

```diff
@@ -291,3 +342,98 @@ def tag_pix_thr(image, thr_low = 50, thr_up = 500, pe_thr = 10):
     """
 
     return ((np.size(image[image > pe_thr]) < thr_up) and (np.size(image[image > pe_thr]) > thr_low))
+
+
+def radial_light_distribution(centre_x, centre_y, pixel_x, pixel_y, image):
+    """
+
+    Parameters
+    ----------
+    centre_x: float
+        Centre of muon ring in the field of view from circle fitting
+    centre_y: float
+        Centre of muon ring in the field of view from circle fitting
+    pixel_x: ndarray
+        X position of pixels in image
+    pixel_y: ndarray
+        Y position of pixel in image
+    image: ndarray
+        Amplitude of image pixels
+
+    Returns
+    -------
+    standard_dev, skewness
+    """
+    # Convert everything to degrees:
+    x0 = centre_x.to(u.deg).value
+    y0 = centre_y.to(u.deg).value
+    pix_x = pixel_x.to(u.deg).value
+    pix_y = pixel_y.to(u.deg).value
+
+    pix_r = np.hypot(pix_x-x0, pix_y-y0)
+
+    # mean, standard deviation & skewness of light distribution along ring radius.
+    # ring_radius calculated elsewhere is approximately equal to "mean", but not exactly, so we recalculate it here:
+    mean = np.sum(np.multiply(image,pix_r))/np.sum(image)
+    standard_dev = np.sqrt(np.sum(np.multiply(image,np.square(pix_r-mean)))/np.sum(image))
```

Comment:
Same as above. numpy and scipy provide weighted averages and standard deviations. Which will both be more readable and faster than this.

Reply:
> Same as above. numpy and scipy provide weighted averages and standard deviations. Which will both be more readable and faster than this.

First thing I intended to do, but failed to find it. Can you point me to those functions?

Reply:
> > Same as above. numpy and scipy provide weighted averages and standard deviations. Which will both be more readable and faster than this.
> 
> First thing I intended to do, but failed to find it. Can you point me to those functions?

ok, saw your previous comment now.

Reply:
I already implemented this in the PR I opened in your fork, I sent you the link by mail

Reply:
https://github.com/moralejo/cta-lstchain/pull/1

Reply:
Done